sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/AppContext.kt

```diff
@@ -153,7 +157,7 @@ object AppContext {
 
     private fun storeAll(networkSponsorJson: String, networkSpeakerJson: String, networkSessionJson: String) {
         appSettings.putString(SPONSOR_JSON, networkSponsorJson)
```

Comment:
I don't think we need to put the sponsors into app settings anymore right?

Reply:
You're correct, I meant to get rid of this when I switched over the iOS side.  It's removed now.